asciidoc-confluence-publisher-doc/etc/docs/00-index.adoc

```diff
@@ -136,6 +141,10 @@ remove all orphaned pages (requires the documentation source structure to contai
 | The message that will be published with the changes into `Page History` comment.
 | optional (defaults to empty)
 
+| attributes| Attributes used during AsciiDoctor processing.
```

Comment:
We would also have to document the format of the attributes for the Docker-based publishing, incl. adjusting the example command in `00-index.adoc`.

Reply:
Update example with command line. Where do you want me to document parameters for docker image?

Reply:
For now, I would just mention in the description of `attributes` that when using Docker-based publishing, user attributes have to be provided as JSON object structure. In the future we might have to separate the configuration parameter documentation for the Maven plugin from the Docker image (some configuration parameters like `skip` do not apply to both mechanisms), but for this pull request, I suggest to keep it simple.